app/src/pages/eligibility.tsx

```diff
@@ -50,11 +50,11 @@ const Eligibility: NextPage<Props> = (props: Props) => {
         route: previousRoute,
       })
     } else setContinueBtn({ ...continueBtn, route: incomeRoute })
-  }, [form.none, props.previousRoute])
+  }, [form.none, props.previousRoute, continueBtn])
 
   useEffect(() => {
     setDisabled(!requiredMet())
-  }, [form])
+  }, [form, requiredMet])
```

Comment:
These dependency changes might be what is causing the tests to hang. Are the tests hanging for you locally?